docs/en/observability/apm/known-issues.asciidoc

```diff
@@ -100,3 +100,33 @@ Failed installing package [apm] due to error: [ResponseError: mapper_parsing_exc
 
 // Link to fix
 A fix was released in 8.11.2: https://github.com/elastic/kibana/pull/171712[elastic/kibana#171712].
+
+
+*APM integration package upgrade through Fleet causes excessive rollovers* +
+_APM Server versions: 7.16.0 to 8.12.1 +
```

Comment:
I don't really like the "7.16" part here. I found that we started using constant_keyword without a value since 7.16, but I am not sure if it actually caused any issues back then. Will `<=8.12.1` or `<8.12.2` be better?

Reply:
updated